### DISCUSSION TITLE
When I was using the case of workflow+knowledge base, I encountered a problem

### DISCUSSION THREAD

**Discussion by Lz020316** [langgenius/dify#11348] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

When I was using the case of workflow+knowledge base, I encountered a problem

Tracking the execution chain of the knowledge base in the workflow, it was found that the result returned by the knowledge base was empty. So, I used the example of this knowledge base node and directly tested it using the recall testing function in the knowledge base. At this time, the result returned by the recall testing is normal. Has a friend encountered this problem before? I don't know why this is happening, I need help,

**Comment by JieJueSama** [comment]
I have also encountered this problem, is there any good solution?
